ciscoasa/objects.go

```diff
@@ -71,16 +71,16 @@ func (s *objectsService) objectFromService(service string) (*ServiceObject, erro
 	o := &ServiceObject{}
 	parts := strings.SplitN(service, "/", 2)
 
-	if len(parts) == 2 && parts[1] != "" && !regexpPorts.MatchString(parts[1]) {
+	if len(parts) == 1 && parts[0] != "" && !regexpPorts.MatchString(parts[0]) {
```

Comment:
This part no longer has any value: `parts[0] != "" &&` as in the beginning of the function we already check: `if service == ""`

So if you can delete that part, the PR is good to go... Thx!